src/pages/have-i-been-pwned.tsx

```diff
@@ -0,0 +1,124 @@
+  const cardsForFeatureSection = [
+    {
+      icon: Globe,
+      title: langJson.FeatureSectionV2.cards![0].title,
+      description: langJson.FeatureSectionV2.cards![0].description,
+    },
+    {
+      icon: FacebookLogo,
+      title: langJson.FeatureSectionV2.cards![1].title,
+      description: langJson.FeatureSectionV2.cards![1].description,
+    },
+    {
+      icon: CheckSquare,
+      title: langJson.FeatureSectionV2.cards![2].title,
+      description: langJson.FeatureSectionV2.cards![2].description,
+    },
+    {
+      icon: Detective,
+      title: langJson.FeatureSectionV2.cards![3].title,
+      description: langJson.FeatureSectionV2.cards![3].description,
+    },
+  ];
+  return (
+    <Layout
+      title={metatags[0].title}
+      description={metatags[0].description}
+      segmentName="Monitor"
+      lang={lang}
+      specialOffer={`https://internxt.com/images/previewLink/LifetimePreviewLink.png`}
+    >
+      <Navbar textContent={navbarLang} lang={lang} cta={['default']} fixed />
+
+      <HeroSection textContent={langJson.HeroSection} />
+
+      <AllGoodSection textContent={langJson.AllGoodSection} />
+
+      <PwnedSection textContent={langJson.PwnedSection} />
+
+      <InfoSection textContent={langJson.InfoSection} />
+
+      <ComponentsInColumnSection
+        FirstComponent={
+          <div className="flex w-full flex-col items-center gap-9">
+            <div className="flex max-w-[774px] flex-col items-center gap-6 text-center">
+              <h2 className="text-5xl font-semibold text-gray-100">{langJson.FeatureSectionV2.title}</h2>
+              <p className="font-regular text-xl text-gray-80">{langJson.FeatureSectionV2.description}</p>
+            </div>
+          </div>
+        }
+        SecondComponent={
+          <div className="flex flex-col items-center space-y-12">
+            <CardGroup cards={cardsForFeatureSection} backgroundColorCard="bg-white" />
+            <SignUpBanner textContent={bannerLang.SignUpPCloudAlternativeBanner} lang={lang} />
+          </div>
+        }
+        backgroundColor="bg-gray-1"
+      />
+
+      <CtaSection
+        textContent={langJson.CtaSection}
+        customDescription={<p className="font-regular text-xl">{langJson.CtaSection.description}</p>}
+        url={''}
+      />
+
+      <FeatureSection textContent={langJson.FeatureSection} />
+
+      <CtaSection
+        textContent={langJson.CtaSection2}
+        customDescription={<p className="font-regular text-xl">{langJson.CtaSection2.description}</p>}
+        url={''}
+      />
+
+      <ToolsSection textContent={toolsContent} lang={lang} />
+
+      <Footer textContent={footerLang} lang={lang} hideNewsletter={false} />
+    </Layout>
+  );
+};
+
+export async function getServerSideProps(ctx) {
+  const lang = ctx.locale;
+
+  const metatagsDescriptions = require(`@/assets/lang/${lang}/metatags-descriptions.json`);
+  const langJson = require(`@/assets/lang/${lang}/i-have-been-pawned.json`);
```

Comment:
You have to create a JSON type in `assets/types` to type the Have I been pawned JSON. Then, you can use smth like HaveIBeenPwnedText in the interface.

Reply:
Done